tests/use_cases/systest_zone_counting.yml

```
@@ -10,7 +10,8 @@ nodes:
     [[0, 0], [0.6, 0], [0.6, 1], [0, 1]],
     [[0.6, 0], [1, 0], [1, 1], [0.6, 1]]
     ]
+- heuristic.fps
 - draw.bbox
 - draw.btm_midpoint
 - draw.zones
-- draw.zone_count
\ No newline at end of file
+- draw.legend
```

```suggestion
- draw.legend

```
new line at end of document